coder/main.go

```diff
@@ -170,9 +187,36 @@ func Run(s *discordgo.Session, m *discordgo.MessageCreate) error {
 	//cli, err := client.NewEnvClient()
 	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
 	defer cancel()
+	
+	//Has attachments files
+	if(len(m.Attachments) > 0){
+		lang:="undefined";
```

Comment:
This should be `nil` instead of "undefined", or empty string (`var lang string`).

However, based on the comment below regarding languages, this should not be required at all.